blog/templates/blog/blogbase.html

```diff
@@ -13,10 +13,10 @@
     {% block title %} {% endblock %} 
 
     <!-- Bootstrap core CSS -->
-    <link href="/static/css/bootstrap.css" rel="stylesheet">
+    <link href="{% static 'bootstrap/css/bootstrap.css' %}" rel="stylesheet">
 
     <!-- Custom styles for this template -->
-    <link href="/static/blog.css" rel="stylesheet">
+    <link href="{% static 'css/blog.css' %}" rel="stylesheet">
 
     <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
     <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
```

Comment:
remove this since you're not actually using it and this path won't be referenced correctly.  This will remove one of the 404 (file not found) errors you're seeing.